index.html

```diff
@@ -5,77 +5,79 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="styles.css" />
-    <link rel="stylesheet" href="assests/fontAwesome/css/all.css" />
     <title>Personal Portfolio</title>
   </head>
   <body>
     <main class="container">
-      <section class="header-div">
-        <header class="header">
-          <section class="toolbar">
-            <a href="index.html" class="mylogo">GeekyHacks</a>
-            <button class="menu-button" type="button" title="Menu">
-              <i class="fa-solid fa-bars fa-fade fa-2x icon-menu"></i>
+      <header class="header">
+        <h2><a href="index.html" class="mylogo">GeekyHacks</a></h2>
+        <nav class="menu-button">
+          <button type="button" title="Menu">
+            <img src="assests/images/menubutton.svg" alt="menu-button" />
+          </button>
+        </nav>
+      </header>
+
+      <section class="headlines">
+        <h3 class="primary">
+          Hey there. I’m Abdullah. <br />
+          <span class="dev-span"> I’m a software developer</span>
+        </h3>
+        <p class="supporting-content">
+          I can help you build a product , feature or website Look through some
```

Comment:
- **[OPTIONAL]** As requested by the last code reviewer please 🙏🏽 ensure that your design match as much as possible with the Figma design, I suggest using `br` tag to "break" the line in the correct position.

| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/228402905-646360c2-d103-44b8-bcb8-79d4048869ba.png) | ![image](https://user-images.githubusercontent.com/58642949/228402998-41b5e568-5a6f-493b-83a4-2a91d50c0d94.png) |